modules/claims_api/spec/lib/claims_api/local_bgs_spec.rb

```diff
@@ -68,13 +68,13 @@
 
         # Events logged:
         # 1: establish_ssl_connection - how long to establish the connection
-        # 2: connection_wsdl_get - duration of WSDL request cycle
-        # 3: connection_post - how long does the post itself take for the request cycle
-        # 4: parsed_response - how long to parse the response
-        expect(ClaimsApi::Logger).to receive(:log).exactly(4).times
+        # 2: connection_post - how long does the post itself take for the request cycle
+        # 3: parsed_response - how long to parse the response
+        expect(ClaimsApi::Logger).to receive(:log).exactly(3).times
         result = subject.find_poa_by_participant_id('does-not-matter')
         expect(result).to be_a Hash
-        expect(result[:end_date]).to eq '08/26/2020'
+        expect(result[:begin_date]).to eq '09/03/2024'
+
```

Comment:
@stiehlrod  I think Rubocop  is going to want to remove this empty line.  When I pulled this down and ran my PR script on it, which runs Rubocop and the test suite, I saw Rubocop actually made a change here because of the empty line.
```
::error file=modules/claims_api/spec/lib/claims_api/local_bgs_spec.rb,line=77,col=1::Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body end.
```

Mentioning it because it may be cleanest to just remove it here instead of the next person that runs Rubocop having this unrelated (most likely) change in their PR.  Looks like rubocop never ran on this because of the CVE failure, but I suspect it will call this line out once it does run (unless. the Rubocop check has changed on Github)

Reply:
@rockwellwindsor-va Thank you!